scene/gui/color_picker.cpp

```diff
@@ -1035,6 +1328,19 @@ void ColorPicker::_preset_input(const Ref<InputEvent> &p_event, const Color &p_c
 	}
 }
 
+void ColorPicker::_recent_preset_pressed(const bool p_pressed, ColorPresetButton *p_preset) {
+	if (!p_pressed) {
+		return;
+	}
+	set_pick_color(p_preset->get_preset_color());
+
+	recent_presets.move_to_back(recent_presets.find(p_preset->get_preset_color()));
+	recent_preset_cache.move_to_back(recent_preset_cache.find(p_preset->get_preset_color()));
```

Comment:
I got another crash around here. I think you should add a check for the result of `find()`, because it returns null if element does not exist.

Reply:
This must be in the scene as `recent_presets` and `recent_preset_cache` contains same elements in the editor.